### DISCUSSION TITLE
Raspberry pi4 8GB vs Jetson Nano for my personal object detection hub?

### DISCUSSION THREAD

**Discussion by 9christian9** [ultralytics/yolov5#2823] [Q&A]
## ❔Question

Ciao guys,
I'm new to this world and I'm searching to learning all about it though it is not very easy...
I'm here to ask you, which Raspberry pi4 8GB vs Jetson Nano I must be chosen to develop a small hub of Object detection?

## Additional context

I should be developing a people detector for my home that monitors about 5/6 cameras in my home h24 7/7
I can connect my cameras using RTSP.

Thanks a lot,
Christian.

**Comment by glenn-jocher** [comment]
@9christian9 Pi lacks a GPU so will be much more resource constrained, but this may be ok if a low framerate is acceptable in your use case. Nano will use more power but provide you more inference capability.

If I were you I would buy both and evaluate them side by side for your needs.

**Reply by 9christian9** [reply]
@glenn-jocher thanks very much for your reply...
So.. I'm searching an hardware that give me much relireliability and much FPS because I will use it to stream online in my personal application to share with my parents...
Now I test it on a mac that is of 2015 and it is very slow, 5FPS....
If I will buy a Raspberry pi4 8gb and I will buy an external GPU?
Pi it is most cheap 😆
What do you suggest?
Can you syggest another Hardware that I can will use?

Sorry for my bad English 🙃

**Reply by glenn-jocher** [reply]
The main edge options are jetson devices (nano, xavier, etc.) and systems with Coral/Google Edge TPUs attached, and if your use case requires very limited resources, i.e. < 5FPS, a Raspberry Pi.
